test/set_test.js

```diff
@@ -0,0 +1,153 @@
+goog.module('datalayerhelper.helper.testing.set');
+goog.setTestOnly();
+
+const {DataLayerHelper} = goog.require('helper');
+
+describe('The set command', function() {
```

Comment:
Please prefer arrow functions, where appropriate:
https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions

I don't see any reference to 'this' in this file, so scoping shouldn't be an issue as it is when invoking "registerProcessor".

Reply:
I have fixed this, thank you for catching that. Just wanted to give a heads up that in beforeAll the command API references the arguments object, so I left it as a regular function.